index.js

```diff
@@ -524,42 +522,37 @@ Lexer.prototype = {
    */
 
   include: function() {
-    var tok = this.scanEndOfLine(/^include +([^\n]+)/, 'include');
+    var tok = this.scan(/^include/, 'include');
     if (tok) {
       this.tokens.push(tok);
+      this.inInclude = true;
+      while (this.inInclude) {
+        this.filter() ||
+          this.attrs() ||
+          this.path() ||
+          this.fail();
+      }
```

Comment:
> Another thought, should we also be emitting two tokens for extends for consistency?

Done in a backward-incompatible way. Before, `extends`, if no argument is present, is treated as a tag. Now, it is consistent with the behavior of `include`.